stdlib/xmlutils/src/test/resources/test-src/xml-to-json-test.bal

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/xmlutils;
+
+function testToJson() returns json|error {
+    var x1 = xml `<!-- outer comment -->`;
+    var x2 = xml `<name>supun</name>`;
+    xml x3 = x1 + x2;
+    json|error j = xmlutils:toJSON(x3, {});
```

Comment:
```suggestion
    json|error j = xmlutils:toJSON(x3);
```

Reply:
This applies iff https://github.com/ballerina-platform/ballerina-lang/pull/18436/files#r320624993 is okay.

Reply:
Added